enos/modules/run_workloads/jobs/docker-service.nomad.hcl

```
@@ -0,0 +1,24 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
```

This suggestion got dropped in the previous pass. We need this for the `var.alloc_count` to work:

```suggestion

variable "alloc_count" {
  type    = number
  default = 1
}

```

---

yeah, I was just informed: 
```Error getting job struct: Error parsing job file from
../../modules/run_workloads/jobs/docker-service.nomad.hcl:
<nil>: Undefined -var variable; A "alloc_count" variable was passed in the
command line but was not found in known variables. To declare variable
"alloc_count", place this block in your job file```